spec/requests/users_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+  describe "GET /index" do
+    pending "add some examples (or delete) #{__FILE__}"
```

Comment:
Since this is a new feature, please add some automated tests!

Reply:
Yes